irods/test/data_obj_test.py

```diff
@@ -2024,11 +2024,12 @@ def test_settings_load_and_save_471(self):
                     config.load(**load_logging_options)
                     self.assertTrue(config.data_objects.auto_close, RANDOM_VALUE - i - 1)
 
+    @unittest.skipIf(os.environ.get("PYTHON_IRODSCLIENT_CONFIGURATION_PATH",None) is not None,"test will not run if configuration file set.")
```

Comment:
Do we want to be sure that this test is run when the full test suite is run? If so, we should make this an assertion at the beginning of the test. If not, the skip is fine.

Reply:
Let's use https://github.com/irods/python-irodsclient/issues/591 for this - it can happen at another time. Resolving.